kubernetes/samples/scripts/create-fmw-infrastructure-domain/domain-home-in-image/create-domain.sh

```diff
@@ -173,50 +173,39 @@ function getDockerSample {
-  passwordFile="${dockerPropsDir}/domain_security.properties"
-
-  rcuPropFile="${dockerPropsDir}/rcu.properties"
-  rcuSecurityPropFile="${dockerPropsDir}/rcu_security.properties"
- 
-  # 12213-domain-home-in-image-wdt uses two properties files for the credentials 
-  if [ ! -f $usernameFile ]; then
-    usernameFile="${dockerPropsDir}/adminuser.properties"
-    passwordFile="${dockerPropsDir}/adminpass.properties"
-  fi
-  
-  sed -i -e "s|myusername|${username}|g" $usernameFile
-  sed -i -e "s|welcome1|${password}|g" $passwordFile
 
-  sed -i -e "s|INFRA08|${rcuSchemaPrefix}|g" $rcuPropFile
-  sed -i -e "s|InfraDB:1521/InfraPDB1.us.oracle.com|${rcuDatabaseURL}|g" $rcuPropFile
+  echo "dumping output of ${domainPropertiesOutput}"
+  cat ${domainPropertiesOutput}
 
-  cp -f ${scriptDir}/common/Dockerfile ${dockerDir}/Dockerfile
-  cp -f ${scriptDir}/common/createFMWDomain.sh ${dockerDir}/container-scripts
+  echo "Invoking WebLogic Image Tool to create a WebLogic domain at '${domainHome}' from image '${domainHomeImageBase}' and tagging the resulting image as '${BUILD_IMAGE_TAG}'."
 
-  if [ ! -z $domainHomeImageBase ]; then
-    sed -i -e "s|\(FROM \).*|\1 ${domainHomeImageBase}|g" ${dockerDir}/Dockerfile
+  echo "fmwDomainType is [${fmwDomainType}]"
+  # fmwDomainType is either JRF or RestrictedJRF. JRF does not support Dynamic Cluster Model
+  if [ "${fmwDomainType}" == "RestrictedJRF" ]; then
+    wdtModelFile=wdt_model_restricted_jrf_configured.yaml
+    wdtDomainType=RestrictedJRF
+  else
+    wdtModelFile=wdt_model_configured.yaml
+    wdtDomainType=JRF
   fi
 
-  cp ${domainPropertiesOutput} ${dockerPropsDir}
-  sed  -i  '$ a extract_env IMAGE_TAG ${PROPERTIES_FILE} ' ${dockerDir}/container-scripts/setEnv.sh
-  sed  -i  '$ a set_env_arg EXPOSE_T3_CHANNEL ${PROPERTIES_FILE} ' ${dockerDir}/container-scripts/setEnv.sh
-  sed  -i  '$ a set_env_arg FMW_DOMAIN_TYPE ${PROPERTIES_FILE} ' ${dockerDir}/container-scripts/setEnv.sh
-  sed  -i  '$ a set_env_arg T3_CHANNEL_PORT ${PROPERTIES_FILE} ' ${dockerDir}/container-scripts/setEnv.sh
-  sed  -i  '$ a set_env_arg T3_PUBLIC_ADDRESS ${PROPERTIES_FILE} ' ${dockerDir}/container-scripts/setEnv.sh
-
-  cp -f ${scriptDir}/common/createFMWDomain.py \
-        ${dockerDir}/container-scripts/createFMWDomain.py
-
-  # Set WDT_VERSION in case dockerDir references a WDT sample
-  # (wdtVersion comes from the inputs file)
-  export WDT_VERSION="${WDT_VERSION:-${wdtVersion:-1.9.1}}"
-  bash ${dockerDir}/build.sh
+  cmd="
+    $WIT_DIR/imagetool/bin/imagetool.sh update
+      --fromImage \"$domainHomeImageBase\"
+      --tag \"${BUILD_IMAGE_TAG}\"
+      --wdtModel \"${scriptDir}/${wdtModelFile}\" 
+      --wdtVariables \"${domainPropertiesOutput}\"
+      --wdtOperation CREATE
+      --wdtVersion ${WDT_VERSION} 
+      --wdtDomainType ${wdtDomainType} 
+      --wdtDomainHome \"${domainHome}\"
+      --additionalBuildCommands additional-build-commands
+      --chown=oracle:oracle
```

Comment:
Can you verify the --chown oracle:oracle works for both fmw and weblogic base image from container-registry.oracle.com?

Reply:
with a weblogic base image, we got this error:
`The Oracle Home directory in --fromImage=container-registry.oracle.com/middleware/weblogic:12.2.1.4 has an owner and group of oracle:root.  You must update the image with the same user and group setting from when it was created. Example: --chown=oracle:root`

But isn't an FMW image required as base image for JRF/RestrictedJRF domain type?

Reply:
Yes, but the two base images fmw and weblogic have different default ownership;  so if this script supports both fmw/weblogic domain types, then the chown parameter input needs to be adjusted.

Reply:
Thanks. This script is for fmw domains only. There is another script for wls domain in kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh which contains:

    $WIT_DIR/imagetool/bin/imagetool.sh update
      --fromImage \"$domainHomeImageBase\"
      --tag \"${BUILD_IMAGE_TAG}\"
      --wdtModel \"${scriptDir}/wdt_model_dynamic.yaml\"
      --wdtVariables \"${domainPropertiesOutput}\"
      --wdtOperation CREATE
      --wdtVersion ${WDT_VERSION}
      --wdtDomainHome \"${domainHome}\" --chown=oracle:root